libs/carrier-portal/types/src/lib/person-view-data.ts

```diff
@@ -145,3 +145,39 @@
   subscriber_hbx_member_id: '',
   status: 'Submitted',
 };
+
+function subscriber(pol: Policy): Enrollee | undefined {
+  return pol.enrollees.find(
+    (en: Enrollee) => en.hbx_member_id == pol.subscriber_hbx_member_id
+  );
+}
+
+export function isCancelled(pol: Policy): boolean {
+  const sub = subscriber(pol);
+  if (sub) {
+    if (sub.coverage_end) {
+      return sub.coverage_end <= sub.coverage_start;
+    }
+  }
+  return false;
+}
+
+export function isTerminated(pol: Policy): boolean {
+  const sub = subscriber(pol);
+  if (sub) {
+    return !!sub.coverage_end;
+  }
+  return false;
+}
+
+export function policyStartDate(policy: Policy): string {
+  const enrollees = policy.enrollees;
+  return enrollees
+    .map((a) => a.coverage_start)
+    .sort()
+    .reverse()[0];
```

Comment:
I'm fully intending to compare these values as strings.

Reply:
You claim:

> the string "2023-02-01" would be considered greater than "2023-12-01" in a string comparison because "02" is lexicographically greater than "12".

Chrome says:
```
"02" > "12"
false
```
Chrome also says:
```
"02".localeCompare("12")
-1
```

ISO8601 was created explicitly to solve the problem you are claiming occurs.

The above statement is factually incorrect.